tutorials/tools/Denormalizaion-Inverse_Text_Normalization.ipynb

```diff
@@ -0,0 +1,325 @@
+    "Note, this tutorial assumes both NeMo and nemo_tools are installed, see [`this`](https://github.com/NVIDIA/NeMo) and [`this`](https://github.com/NVIDIA/NeMo/tree/main/nemo_tools) for details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "homeless-richardson",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BRANCH = 'main'\n",
+    "\n",
+    "import os\n",
+    "import pynini"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "interracial-listing",
+   "metadata": {},
+   "source": [
+    "# Task Description\n",
+    "\n",
+    "Inverse text normalization (ITN), also called denormalization, is a part of the Automatic Speech Recognition (ASR) post-processing pipeline. \n",
+    "\n",
+    "ITN is the task of converting the raw spoken output of the ASR model into its written form to improve the text readability. For example, `in nineteen seventy` should be changed to `in 1975` and `one hundred and twenty three dollars` to `$123`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "desirable-words",
+   "metadata": {},
+   "source": [
+    "# NeMo Inverse Text Normalization\n",
+    "\n",
+    "The NeMo ITN tool is a Python package that is based on weighted finite-state\n",
+    "transducer (WFST) grammars. The tool uses [`Pynini`](https://github.com/kylebgorman/pynini) to construct WFSTs, and the created grammars can be exported and integrated into [`Sparrowhawk`](https://github.com/google/sparrowhawk) (an open source version of [The Kestrel TTS text normalization system](https://www.cambridge.org/core/journals/natural-language-engineering/article/abs/kestrel-tts-text-normalization-system/F0C18A3F596B75D83B75C479E23795DA)) for production. The NeMo ITN tool is a Python extension of `Sparrowhawk`. \n",
+    "\n",
+    "Currently, NeMo tool provides support for English language and the following semiotic classes from the [Google Text normalization dataset](https://www.kaggle.com/richardwilliamsproat/text-normalization-for-english-russian-and-polish):\n",
+    "DATE, CARDINAL, MEASURE, DECIMAL, ORDINAL, MONEY, PLAIN. \n",
+    "\n",
+    "The toolkit is modular, easily extendable, and can be adapted to other languages and tasks like text normalization. The Python environment enables easy combination of text covering grammars with NNs. \n",
+    "\n",
+    "The overall NeMo ITN pipeline from development in `Pynini` to deployment in `Sparrowhawk` is shown below:\n",
+    "![alt text](deployment.png \"Inverse Text Normalization Pipeline\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "military-radius",
+   "metadata": {},
+   "source": [
+    "# Quick Start\n",
+    "\n",
+    "## Add ITN to your Python ASR post-processing workflow:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "limiting-genesis",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nemo_tools.text_denormalization.denormalize import denormalize\n",
```

Comment:
emphasize that ITN is part of package nemo_tools, which can be installed as shown in main README